e.rs

```diff
@@ -1,7 +1,15 @@
-#![feature(main)]
-#[main]
```

Comment:
I would like to keep `#[feature(main)]` and marking `fn e` as `#[main]` because it makes me giggle.